silly/PKGBUILD

```diff
@@ -4,24 +4,32 @@
 
 pkgname=silly
 pkgver=0.1.0
-pkgrel=8
+pkgrel=9
 pkgdesc="Simple Image Loading LibrarY, a part of the CEGUI project"
-arch=('x86_64')
+arch=('x86_64' 'aarch64')
```

Comment:
```suggestion
arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
```

Reply:
Let's have em all!

Reply:
Ahh! totally forgot about .SRCINFO updated it and added all the arch's